tensorflow/compiler/xla/stream_executor/rocm/rocm_dnn.cc

```diff
@@ -3868,6 +3871,155 @@ bool MIOpenSupport::DoBatchNormalizationBackwardImpl(
   return true;
 }
 
+template <typename T>
+void launchInplaceBiasActivation(hipStream_t stream, void* c_data, const void* bias_data, int activation_mode, uint64_t m, uint64_t n, int64_t ldc, float param);
```

Comment:
There seem to be quite a lot of long lines in this file. Can you please reformat to lines of length <= 80?